Adyen/Service/Checkout/OrdersService.cs

```diff
@@ -34,18 +34,58 @@ public OrdersService(Client client) : base(client)
             _baseUrl = client.Config.CheckoutEndpoint + "/" + ClientConfig.CheckoutApiVersion;
         }
     
+        public CheckoutCreateOrderResponse Orders(CheckoutCreateOrderRequest checkoutCreateOrderRequest, RequestOptions requestOptions = default)
+        {
+            return OrdersAsync(checkoutCreateOrderRequest, requestOptions).GetAwaiter().GetResult();
+        }
+
+        public async Task<CheckoutCreateOrderResponse> OrdersAsync(CheckoutCreateOrderRequest checkoutCreateOrderRequest, RequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            var endpoint = _baseUrl + "/orders";
+            var resource = new ServiceResource(this, endpoint);
+            return await resource.RequestAsync<CheckoutCreateOrderResponse>(checkoutCreateOrderRequest.ToJson(), requestOptions, new HttpMethod("POST"), cancellationToken);
+        }
+
+        public CheckoutCancelOrderResponse CancelOrder(CheckoutCancelOrderRequest checkoutCancelOrderRequest, RequestOptions requestOptions = default)
+        {
+            return CancelOrderAsync(checkoutCancelOrderRequest, requestOptions).GetAwaiter().GetResult();
+        }
+
+        public async Task<CheckoutCancelOrderResponse> CancelOrderAsync(CheckoutCancelOrderRequest checkoutCancelOrderRequest, RequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            var endpoint = _baseUrl + "/orders/cancel";
+            var resource = new ServiceResource(this, endpoint);
+            return await resource.RequestAsync<CheckoutCancelOrderResponse>(checkoutCancelOrderRequest.ToJson(), requestOptions, new HttpMethod("POST"), cancellationToken);
+        }
+
+        public CheckoutBalanceCheckResponse GetBalanceOfGiftCard(CheckoutBalanceCheckRequest checkoutBalanceCheckRequest, RequestOptions requestOptions = default)
+        {
+            return GetBalanceOfGiftCardAsync(checkoutBalanceCheckRequest, requestOptions).GetAwaiter().GetResult();
+        }
+
+        public async Task<CheckoutBalanceCheckResponse> GetBalanceOfGiftCardAsync(CheckoutBalanceCheckRequest checkoutBalanceCheckRequest, RequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            var endpoint = _baseUrl + "/paymentMethods/balance";
+            var resource = new ServiceResource(this, endpoint);
+            return await resource.RequestAsync<CheckoutBalanceCheckResponse>(checkoutBalanceCheckRequest.ToJson(), requestOptions, new HttpMethod("POST"), cancellationToken);
+        }
+
+    }
+    
+    /// <summary>
+    /// Service Interface
+    /// </summary>
+    public interface IOrdersService
+    {
         /// <summary>
         /// Create an order
         /// </summary>
         /// <param name="idempotencyKey">A unique identifier for the message with a maximum of 64 characters (we recommend a UUID).</param>
         /// <param name="checkoutCreateOrderRequest"></param>
```

Comment:
- [ ] We should move the comments to Interface level only (that's what the developers will use/see)
- [ ] Point of improvement: We could annotate the class properl instead of leaving the param empty
```csharp
/// <param name="checkoutCreateOrderRequest"><see cref="CheckoutCreateOrderRequest"/>.</param>
```
- [ ] Point of improvement 2: We could do the same for the return type
```csharp
/// <returns ><see cref="CheckoutCreateOrderResponse"/>.</returns>
```

Reply:
I did move comments to interface, bit hard to see bc of this messed up git diff :)